src/api/app/components/comment_component.rb

```diff
@@ -22,10 +22,21 @@ def attributes
   end
 
   def body
+    return inline_comment_body if comment.commentable.is_a?(BsRequestAction)
+
     comment.body.delete("\u0000")
   end
 
   private
 
   attr_accessor :comment
+
+  def inline_comment_body
+    target = "#{comment.commentable.target_project}/#{comment.commentable.target_package}"
+    file_index, line_number = comment.diff_ref.match(/diff_([0-9]+)_n([0-9]+)/).captures
+    diffs = comment.commentable.bs_request.webui_actions(action_id: comment.commentable, diffs: true, cacheonly: 1)
+    filename = diffs.first[:sourcediff].first['filenames'][file_index.to_i]
+
+    "Inline comment for target: '#{target}', file: '#{filename}', and line: #{line_number}:\n\n#{comment.body}"
```

Comment:
This creates a `.body` -> `.inline_comment_body`  loop no?

Reply:
True. I guess should should remove the unicode null byte then too? I have no idea why we do this though...

Reply:
> True. I guess should should remove the unicode null byte then too? I have no idea why we do this though...

I remember that one: https://github.com/openSUSE/open-build-service/pull/5870

Reply:
2018, those where the days!